cclient/api/extended/send_trytes.c

```diff
@@ -56,19 +56,13 @@ retcode_t iota_client_send_trytes(iota_client_service_t const* const serv, hash8
   hash_array_free(attach_req->trytes);
   attach_req->trytes = trytes;
 
-  get_transactions_to_approve_req_free(&tx_approve_req);
```

Comment:
it's on purpose.
https://github.com/iotaledger/entangled/pull/845#pullrequestreview-207755084